src/SourceCode.Clay.Tests/NumberTests.cs

```diff
@@ -30,11 +30,15 @@ public class NumberTests
             InlineData(nameof(Single) + "=0", 0.0F, NumberKind.Real | NumberKind.Signed, true),
             InlineData(nameof(Single), 9.0F, NumberKind.Real | NumberKind.Signed, false),
             InlineData(nameof(Double) + "=0", 0.0, NumberKind.Real | NumberKind.Signed, true),
-            InlineData(nameof(Double), 10.0, NumberKind.Real | NumberKind.Signed, false)
+            InlineData(nameof(Double), 10.0, NumberKind.Real | NumberKind.Signed, false),
+            InlineData(nameof(Decimal), 10.0, NumberKind.Real | NumberKind.Signed, false)
         ]
         [Theory(DisplayName = nameof(Number_ContructGet))]
         public void Number_ContructGet(string description, object expected, NumberKind kind, bool isZero)
         {
+            if (description == nameof(Decimal))
```

Comment:
Decimal literals can't be used in attributes, so we need to convert in the test.

Reply:
Else maybe create actual `const` decimals in class